exonum/src/proto/mod.rs

```diff
@@ -70,3 +70,27 @@ impl ProtobufConvert for ValidatorId {
         Ok(ValidatorId(pb as u16))
     }
 }
+
+// Think about bincode instead of protobuf. [ECR-3222]
+#[macro_export]
+macro_rules! impl_binary_key_for_binary_value {
```

Comment:
To be honest, I still don't think that it's an appropriate place ^_^" It's all about merkledb, if I understand it correctly, then why to have it in the `proto` module?

Wouldn't it be logical to keep it inside merkledb itself?